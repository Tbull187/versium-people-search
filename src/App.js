import React from 'react';
import logo from './logo.svg';
import { Container, Row, Col } from 'react-bootstrap';
import SearchFormContainer from './components/SearchFormContainer';
import './styles/App.css';

function App() {
  return (
    <div className="App">
      <Container>
        <Row>
          <Col><h1>Welcome to Versium People Search</h1></Col>
        </Row>

        <SearchFormContainer />

      </Container>
    </div>
  );
}

export default App;
