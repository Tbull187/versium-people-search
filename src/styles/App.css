/* searchForm Component */
.btn-row .col {
  padding-bottom: 15px;
}

/* SearchResult Component */
.spinner-container {
  text-align: center;
  padding-top: 20px;
}

.result {
  padding: 10px 15px;
}