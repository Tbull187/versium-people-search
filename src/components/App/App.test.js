import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('renders App component', () => {
  const component = renderer.create(<App />).toJSON();
  expect(component).toMatchSnapshot();
});
