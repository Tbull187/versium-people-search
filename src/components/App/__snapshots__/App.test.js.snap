// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="text-align-center row"
    >
      <div
        className="col"
      >
        <h1>
          Welcome to Versium People Search
        </h1>
      </div>
    </div>
    <div
      className="btn-row row row-cols-lg-4 row-cols-1"
    >
      <div
        className="col"
      >
        <div
          className="input-group"
        >
          <div
            className="input-group-prepend"
          >
            <span
              className="input-group-text"
              id="fist-name-label"
            >
              First Name
            </span>
          </div>
          <input
            aria-describedby="fist-name-label"
            aria-label="First Name"
            className="form-control"
            disabled={false}
            onChange={[Function]}
            placeholder="First Name"
            value=""
          />
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="input-group"
        >
          <div
            className="input-group-prepend"
          >
            <span
              className="input-group-text"
              id="last-name-label"
            >
              Last Name
            </span>
          </div>
          <input
            aria-describedby="last-name-label"
            aria-label="Last Name"
            className="form-control"
            disabled={false}
            onChange={[Function]}
            placeholder="Last Name"
            value=""
          />
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="input-group"
        >
          <div
            className="input-group-prepend"
          >
            <span
              className="input-group-text"
              id="state-label"
            >
              State
            </span>
          </div>
          <input
            aria-describedby="state-label"
            aria-label="State"
            className="form-control"
            disabled={false}
            onChange={[Function]}
            placeholder="State"
            value=""
          />
        </div>
      </div>
      <div
        className="col-md-auto"
      >
        <button
          className="btn btn-primary"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Search
        </button>
      </div>
    </div>
    <div
      className="search-results"
    />
  </div>
</div>
`;
